### DISCUSSION TITLE
不想用dayjs，也不想用任何时间lib，可以吗？

### DISCUSSION THREAD

**Discussion by zzzgit** [vueComponent/ant-design-vue#7104] [General]
https://www.antdv.com/docs/vue/replace-date-cn

我不想用任何工具类库，可以吗？一定要在`dayjs`  `moment ` `date-fns`里面三选一吗？

**Comment by zzzgit** [comment]
同志們，有人能解答嗎？

**Comment by EvanZhang1991** [comment]
可以，源码里有对三个的支持

**Comment by zzzgit** [comment]
這...

**Comment by shifeng1993** [comment]
可以，按照文档，按需引入。

使用时间库的组件有date-picker和time-picker，不引入这俩就可以
